R/utilities.R

```diff
@@ -60,15 +60,14 @@ updateScaleArgumentsForTimeUnit <- function(scale.args,
       if (2 * width > diff(range(x))) {
         return(scales::breaks_extended(n = 5)(x))
       } else {
-        while (20 * width < diff(range(x))) {
+        while (10 * width < diff(range(x))) {
```

Comment:
the comment in the function header does not match the new constant (10).
An explanation what this 10 means would be nice as well